kyuubi-server/src/main/scala/org/apache/kyuubi/operation/BatchJobSubmission.scala

```diff
@@ -133,6 +133,16 @@ class BatchJobSubmission(session: KyuubiBatchSessionImpl, batchRequest: BatchReq
     }
   }
 
+  def getOperationLogRowSet(
+      order: FetchOrientation,
+      from: Int,
+      size: Int): TRowSet = {
+    val operationLog = getOperationLog
+    operationLog.map(_.read(from, size)).getOrElse {
```

Comment:
It seems that, if the from < 0, we can invoke `def read(maxRows: Int): TRowSet` to read forward and `seekableReader` is not needed.

Reply:
I considered the hybrid case that the first request is [-1, 100), the second request is [100, 110). So it would have better performance if we always use seekable reader.

Reply:
got it